README.md

```diff
@@ -1,4 +1,4 @@
-[![mailchimp3 v3.0.9 on PyPi](https://img.shields.io/pypi/v/mailchimp3.svg)](https://pypi.python.org/pypi/mailchimp3)
+[![mailchimp3 v3.0.10 on PyPi](https://img.shields.io/pypi/v/mailchimp3.svg)](https://pypi.python.org/pypi/mailchimp3)
```

Comment:
Change: 3.0.10 is the current version, please update to 3.0.11

Reply:
Sorry, I should have noticed the changes. I have updated the version to 3.0.11